nuxt.config.js

```diff
@@ -21,7 +21,7 @@ export default {
       { name: 'format-detection', content: 'telephone=no' }
     ],
     link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+      { rel: 'icon', type: 'image/png', href: '/logo.png' }
```

Comment:
Png is supported by browsers? 🤔

Reply:
Yes it is since fucking 2010 https://caniuse.com/?search=png%20icon